File: Common/Entity/src/main/java/it/ey/dto/PaRiferimentoDTO.java
package it.ey.dto;

import lombok.*;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class PaRiferimentoDTO {

    private boolean attiva;
    private String externalId;
    private String codePA;
    private String fiscalCode;
    private String email;
    private String numeroTelefono;
    private String qualifica;
    private String denominazionePA;
    private List<RuoloUserDTO> ruoli;

}

File: dfp-piao-fe/src/app/shared/models/classes/pa-riferimento-dto.ts
import { RuoloUtenteDTO } from './ruolo-utente-dto';

export class PARiferimentoDTO {
  attiva!: boolean;
  externalId?: string;
  codePA!: string;
  email?: string;
  fiscalCode?: string;
  numeroTelefono?: string;
  qualifica?: string;
  denominazionePA!: string;
  ruoli!: RuoloUtenteDTO[];
}
